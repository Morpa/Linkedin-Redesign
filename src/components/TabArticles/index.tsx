import * as S from './styles'

const TabArticles = () => (
  <S.Wrapper>
    <h1>TabArticles</h1>
  </S.Wrapper>
)

export default TabArticles
