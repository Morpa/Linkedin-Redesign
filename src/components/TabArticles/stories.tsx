import { Story, Meta } from '@storybook/react/types-6-0'
import TabArticles from '.'

export default {
  title: 'TabArticles',
  component: TabArticles
} as Meta

export const Default: Story = () => <TabArticles />
