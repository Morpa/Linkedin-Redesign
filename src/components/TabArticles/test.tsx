import { render, screen } from '@testing-library/react'

import TabArticles from '.'

describe('<TabArticles />', () => {
  it('should render the heading', () => {
    const { container } = render(<TabArticles />)

    expect(
      screen.getByRole('heading', { name: /TabArticles/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
