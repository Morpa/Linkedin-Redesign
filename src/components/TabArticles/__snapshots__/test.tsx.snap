// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabArticles /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    TabArticles
  </h1>
</main>
`
